requirements/base-requirements.in

```diff
@@ -42,6 +42,7 @@ elasticsearch2>=2.0.0,<3.0.0  # we will be deprecating support for ES2 soon
 elasticsearch5>=5.0.0,<6.0.0  # ES5 support is currently in development
 ethiopian-date-converter
 eulxml
+firebase-admin
```

Comment:
for all the dependencies being added, can you please add a comment next to the dependency with the division owning this dependency and what feature(s) it's tied to? Thanks!

Reply:
Sure . Thanks @biyeun

Reply:
Resolved in https://github.com/dimagi/commcare-hq/pull/33047/commits/2862e4a02806f9e74c703c1b8a3a9f1c99c6edf8